### PR TITLE
Updating TPN to the current MSFT format and adding Humanizer, SQLitePCLRaw and ICSharpCode.Decompiler

### PR DESCRIPTION
**Pull request by ivanbasov**

### CODE REVIEW

**Comment by jasonmalinowski** [comment]
Are we also missing the ILSpy third party notification?

**Comment by ivanbasov** [comment]
> Are we also missing the ILSpy third party notification?

Thank you, @jasonmalinowski ! Do we distribute it with our source code or with packages?

**Comment by jasonmalinowski** [comment]
We use it the same way as Humanizer...

**Comment by ivanbasov** [comment]
> We use it the same way as Humanizer...

@jasonmalinowski not sure we re-distribute it as we do with Humanizer. Let us sync offline.

**Comment by ivanbasov** [comment]
@jasonmalinowski , @sharwell 
Please review:
1. removed humanizer from source code tpn
2. added humanizer, icsharpcode.decompiler , sqlitepclraw.bundle_green , sqlitepclraw.core , sqlitepclraw.lib.e_sqlite3.v110_xp  and sqlitepclraw.provider.e_sqlite3.net45 to vsix tpn
3. added eula and tpn to all vsix packages
4. added eula as a license to all vsix packages

File: eng/targets/VisualStudio.targets [hunk 1]
```diff
@@ -125,4 +125,19 @@
...
+    <Content Include="$(MSBuildThisFileDirectory)\..\..\src\Setup\Roslyn.VsixLicense\EULA.rtf">
+      <Link>Eula.rtf</Link>
+      <IncludeInVSIX>true</IncludeInVSIX>
+    </Content>
+    <Content Include="$(MSBuildThisFileDirectory)\..\..\src\Setup\Roslyn.ThirdPartyNotices\ThirdPartyNotices.rtf">
+      <Link>ThirdPartyNotices.rtf</Link>
+      <IncludeInVSIX>true</IncludeInVSIX>
+    </Content>
+
+    <PackageReference Include="Microsoft.DiaSymReader.Native" Version="$(MicrosoftDiaSymReaderNativeVersion)" ExcludeAssets="all"/>
```
Comment (sharwell) [inline eng/targets/VisualStudio.targets]: 😕 Why is this getting added here?

File: eng/targets/VisualStudio.targets [hunk 2]
```diff
@@ -125,4 +125,19 @@
       </ReferencePath>
     </ItemGroup>
   </Target>
+
+  <!-- Add License and Third Party Notices files into each VSIX. -->
+  <ItemGroup Condition="'$(CreateVsixContainer)' == 'true'">
+    <Content Include="$(MSBuildThisFileDirectory)\..\..\src\Setup\Roslyn.VsixLicense\EULA.rtf">
+      <Link>Eula.rtf</Link>
```
Comment (sharwell) [inline eng/targets/VisualStudio.targets]:
```suggestion
      <Link>EULA.rtf</Link>
```

File: src/Compilers/Extension/source.extension.vsixmanifest [hunk 3]
```diff
@@ -4,6 +4,7 @@
     <Identity Id="7922692f-f018-45e7-8f3f-d3b7c0262841" Version="|%CurrentProject%;GetVsixVersion|" Language="en-US" Publisher="Microsoft" />
     <DisplayName>Roslyn Compilers</DisplayName>
     <Description xml:space="preserve">Package of the Roslyn compilers that enables them for Visual Studio builds.</Description>
+    <License>Eula.rtf</License>
```
Comment (sharwell) [inline src/Compilers/Extension/source.extension.vsixmanifest]: 📝 It seems like the case here should match the file referenced in the targets file

```suggestion
    <License>EULA.rtf</License>
```

Comment (ivanbasov) [reply src/Compilers/Extension/source.extension.vsixmanifest]: Updated this in all locations. Thank you!

**Pull request merged**